monitoring/rpc_stats_interceptor.go

```diff
@@ -0,0 +1,95 @@
+package monitoring
+
+import (
+	"expvar"
+	"fmt"
+	"time"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"github.com/google/trillian/util"
+)
+
+const (
+	nanosToMillisDivisor int64 = 1000000
```

Comment:
Yes for readability, no for it remaining a constant as you can't divide Durations. You have to call .Nanoseconds() on them.